web/docs/auth/auth-hooks.md

```diff
@@ -191,7 +197,7 @@ Wasp calls the `onAfterSignup` hook after the user is created.
 
 The `onAfterSignup` hook can be useful if you want to send the user a welcome email or perform some other action after the user signs up like syncing the user with a third-party service.
 
-Since the `onAfterSignup` hook receives the OAuth access token, it can also be used to store the OAuth access token for the user in your database.
+Since the `onAfterSignup` hook receives the OAuth tokens, it can also be used to store the OAuth access token and/or [refresh token](#refreshing-the-oauth-access-token) in your database.
```

Comment:
Passive voice. Please fix in other places as well.

Reply:
I keep writing using passive voice!